### DISCUSSION TITLE
ESLint의 import 정렬

### DISCUSSION THREAD

**Discussion by baechanghyeon** [WantedPreonboardingFE5team/pre-onboarding-7th-2-1-5#32] [Reference]
혹시 앞서 설명드린 절대경로 alias 설정과 지난번에 소개해드린 styleLint쪽 말고도 
혹시 import도 컨벤션을 정하고 자동 정렬을 원하시는 분 있으신가요.....?
최근에 알게된거지만 import도 가능하네요! 혹시 관심 있으신분은 한번 보고 적용해보세요!
참고 URL : [import ESLint 적용 방법](https://db2dev.tistory.com/entry/ESLint-importorder-%EA%B7%9C%EC%B9%99-%EC%84%A4%EC%A0%95%ED%95%98%EA%B3%A0-%EB%92%A4%EC%A3%BD%EB%B0%95%EC%A3%BD-import-%EC%BD%94%EB%93%9C-%EA%B0%9C%EC%84%A0%ED%95%98%EA%B8%B0) 

ESLint 끝은 어디인가...🫠

**Comment by hyoniiii** [comment]
오 대박! 업무 자동화도 끝이 없네요..

**Comment by jaeyeong815** [comment]
ㅋㅋㅋ 저 import 엄청 신경쓰는데 이번 과제하면서는 신경을 못썼네요..
라이브러리, 컴포넌트, 스타일 보통 이렇게 나누는데 자동으로해주면 짱좋겠군여 ㅎ

**Comment by yeondooo** [comment]
와우..한 번 할 때가 귀찮지만 한번 해 놓으면 엄청 편할거 같긴 하네요..! 이번 과제에 도전....?

**Reply by jaeyeong815** [reply]
오늘..alias 설정이랑 import설정 도전..!?!?!?

**Reply by baechanghyeon** [reply]
전에 말씀드렸던 styleLint 도 적용하려고 했지만 설정이 무지 까다롭고, 자료가 별로 없더라구요 ! 
이번과제에 절대경로 import 까지는 괜찮은거 같습니다 ! 그러 boiler plate에 적용시키면 될려나요?

**Reply by yeondooo** [reply]
이번 과제 리포지토리 만들어놓으면서 boilerplate 리포지토리 yarn => npm 변경, 창현님이 공유해주신 alias 설정 해봤습니다..! 기본 폴더에 context 추가했습니다..! https://github.com/WantedPreonboardingFE5team/Boilerplate/commit/c8064a5682ca184f4552b73c3580ec9e8a962102

**Reply by baechanghyeon** [reply]
말보다 행동이네요 연주님... 존경합니다 ㅋㅋㅋㅋㅋ👍
